crates/pkgcraft-tools/src/subcmds/pkg/source.rs

```diff
@@ -338,32 +337,31 @@ fn benchmark(bench: Bench, targets: PkgTargets, cmd: &Command) -> anyhow::Result
                 let mean = Duration::from_micros(mean);
 
                 let value = Benchmark::new(pkg, mean, min, max, sdev, n);
-                if let Some(values) = sorted.as_mut() {
-                    values.push(value);
```

Comment:
Here it is already collecting, I can switch to a for_each

Reply:
That's only when sorting and isn't done by default for that reason.